packages/core/src/components/tabs/tabs.tsx

```diff
@@ -0,0 +1,146 @@
+import { type ComponentPropsWithoutRef, forwardRef } from 'react';
+
+import { Tabs as BaseTabs } from '@base-ui-components/react';
+import clsx from 'clsx';
+
+import { createContext } from '~/libs/create-context';
+import { createSplitProps } from '~/utils/create-split-props';
+
+import * as styles from './tabs.css';
+import type { ButtonVariants, ListVariants } from './tabs.css';
+
+type TabsVariants = ListVariants & ButtonVariants;
+type SharedProps = TabsVariants & { activateOnFocus?: boolean; loop?: boolean };
+type TabsContext = SharedProps;
```

Comment:
There’s no functional issue, but how about changing it as follows for consistency with the other components?
```suggestion
type TabsSharedProps = TabsVariants & { activateOnFocus?: boolean; loop?: boolean };
type TabsContext = TabsSharedProps;
```